codegen/projections/white_label/spec/relative_middleware_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+module WhiteLabel
+  describe Client do
+    context 'relative middleware ordering' do
+      it 'allows a middleware to be set before their relative' do
+        config = Config.new(
+          stub_responses: true,
+          verify_before_middleware: proc { |metadata|
+            expect(metadata[:before_middleware]).to be(true)
+          }
+        )
+        client = Client.new(config)
+        client.request_compression_operation({ body: 'foo' })
```

Comment:
I think request compression operation should only be for testing that - I would use a new test middleware operation here.

Reply:
Added!